lib/ansible/module_utils/basic.py

```diff
@@ -2329,17 +2329,9 @@ def run_command(self, args, check_rc=False, close_fds=True, executable=None, dat
                 cmd.stdin.close()
 
             while True:
-                rfd, wfd, efd = select.select(rpipes, [], rpipes, 1)
-                if cmd.stdout in rfd:
-                    dat = os.read(cmd.stdout.fileno(), 9000)
-                    stdout += dat
-                    if dat == b(''):
-                        rpipes.remove(cmd.stdout)
-                if cmd.stderr in rfd:
-                    dat = os.read(cmd.stderr.fileno(), 9000)
-                    stderr += dat
-                    if dat == b(''):
-                        rpipes.remove(cmd.stderr)
+                rfds, _, _ = select.select(rpipes, [], rpipes, 1)
```

Comment:
We try to avoid defining ```_``` in case we ever decide to i18n the code.  gettext traditionally uses ```_()``` to mark strings for translation and we wouldn't want to overwrite that.